<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-TAVTASOM" failures="1" tests="13" name="Tests.PdamTest" time="137.247" errors="0" timestamp="2021-03-25T12:19:28 WIB" skipped="0">
  <testcase classname="Tests.PdamTest" name="goToLoginPage" time="3.562"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="testLogin" time="0.377"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="goToPdamTest" time="7.475"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="pilihNamaPDAM" time="0.125"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="inputTextPDAM" time="0.178"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="selectInputTextPDAM" time="0.081"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="btnCancelIsDisplay" time="0.063">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Pages.PdamPages.btnCancelIsDisplay(PdamPages.java:87)
at Tests.PdamTest.btnCancelIsDisplay(PdamTest.java:100)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- btnCancelIsDisplay -->
  <system-out/>
  <testcase classname="Tests.PdamTest" name="inputCustomerNoPDAM" time="0.101"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="goToCheckoutPDAM" time="0.078"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="checkOutPDAM" time="3.810"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="goToPaymentCenter" time="0.070"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="PilihAlatBayar" time="120.358"/>
  <system-out/>
  <testcase classname="Tests.PdamTest" name="clickBayar" time="0.969"/>
  <system-out/>
</testsuite> <!-- Tests.PdamTest -->
